### PR TITLE
Setting typescript to version 4.9.5 and adding npm packages

### PR DESCRIPTION
**Pull request by craigbreaden**

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -5,8 +5,14 @@ RUN apt-get update && apt-get install -y rustc cargo nodejs npm python3-sphinx p
 # Install required Sphinx extensions for the documentation
 RUN pip3 install furo sphinx-inline-tabs
 
+# Install required Sphinx extensions for the documentation
```
Comment (garretfick) [inline .devcontainer/Dockerfile:8]: I think there is a duplicated line here.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (garretfick) [inline package.json:1]: Do you know why this is showing in the root of the project? I think this should only be in `integrations/vscode` or perhas the `.devcontainer` folder.

**Review by garretfick — CHANGES_REQUESTED** [review]

File: .devcontainer/Dockerfile [hunk 3]
```diff
@@ -5,8 +5,14 @@ RUN apt-get update && apt-get install -y rustc cargo nodejs npm python3-sphinx p
...
+RUN pip3 install furo sphinx-inline-tabs
+
 # Install just, a task runner
-RUN cargo install --root /usr/local just cargo-release llvm-conv
+RUN cargo install --root /usr/local just cargo-release 
+
+# Install llmv-conv
+RUN cargo install cargo-llvm-cov --locked
```
Comment (garretfick) [inline .devcontainer/Dockerfile:15]: Locked is not necessary here. I had the wrong name, but there is no cargo.toml file so it won't find the version in any case.

**Review by garretfick — CHANGES_REQUESTED** [review]

File: .devcontainer/Dockerfile (hunk 1 above)
Comment (craigbreaden) [reply .devcontainer/Dockerfile]: Resolved.

File: package.json (hunk 2 above)
Comment (craigbreaden) [reply package.json]: During my debugging I had to manually install these packages due to missing deps; not sure how it got to the root folder but I moved it into integrations/vscode and re-tested. Looks to be working.

File: .devcontainer/Dockerfile (hunk 3 above)
Comment (craigbreaden) [reply .devcontainer/Dockerfile]: Removed; blindly following online instructions to get round an error during the build. could probably have moved it back to line 12.

Comment (craigbreaden) [reply .devcontainer/Dockerfile]: In fact, I'll do that now.

**Review by garretfick — APPROVED** [review]

**Pull request merged**